<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>ParseFunction</class>
  <widget class="QDialog" name="ParseFunction">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>620</width>
      <height>600</height>
     </rect>
    </property>
    <property name="windowTitle">
      <string>Parse Function</string>
    </property>
    <widget class="QWidget" name="centralWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout1">
            <item>
              <widget class="QLabel" name="label_FolderFile">
                <property name="text">
                  <string>Folder/File</string>
                </property>
                <property name="minimumWidth">
                 <number>100</number>
                </property>
                <property name="maximumWidth">
                 <number>100</number>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QLineEdit" name="edit_FolderFile">
               <property name="minimumWidth">
                <number>400</number>
               </property>
               <property name="maximumWidth">
                <number>400</number>
               </property>
             </widget>
            </item>
            <item>
              <widget class="QPushButton" name="button_Browse">
                <property name="text">
                  <string>Browse</string>
                </property>
                <property name="minimumWidth">
                 <number>100</number>
                </property>
                <property name="maximumWidth">
                 <number>100</number>
                </property>
              </widget>
            </item>
          </layout>
        </item>

        <item>
          <widget class="QFrame" name="split_line1">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>620</width>
              <height>1</height>
             </rect>
            </property>
            <property name="frameShape">
              <enum>QFrame::HLine</enum>
            </property>
            <property name="FrameShadow">
              <enum>QFrame::Plain</enum>
            </property>
          </widget>
        </item>

        <item>
          <layout class="QHBoxLayout" name="horizontalLayout4">
            <item>
             <widget class="QCheckBox" name="checkBracket">
              <property name="text">
                <string>Use '{}'</string>
              </property>
             </widget>
            </item>
            <item>
              <widget class="QPushButton" name="button_Load">
                <property name="text">
                  <string>Load</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>

        <item>
          <widget class="QFrame" name="split_line2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>620</width>
              <height>1</height>
             </rect>
            </property>
            <property name="frameShape">
              <enum>QFrame::HLine</enum>
            </property>
            <property name="FrameShadow">
              <enum>QFrame::Plain</enum>
            </property>
          </widget>
        </item>

        <item>
          <widget class="QTableWidget" name="tableFunctions">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>620</width>
              <height>540</height>
             </rect>
            </property>
          </widget>
        </item>

        <item>
          <widget class="QFrame" name="split_line3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>620</width>
              <height>1</height>
             </rect>
            </property>
            <property name="frameShape">
              <enum>QFrame::HLine</enum>
            </property>
            <property name="FrameShadow">
              <enum>QFrame::Plain</enum>
            </property>
          </widget>
        </item>

        <item>
          <layout class="QHBoxLayout" name="horizontalLayout4">
            <item>
             <widget class="QCheckBox" name="checkWhole">
              <property name="text">
                <string>Whole File First</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkWriteBack">
              <property name="text">
                <string>Write back(dangers)</string>
              </property>
             </widget>
            </item>
          </layout>
        </item>

        <item>
          <widget class="QFrame" name="split_line4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>620</width>
              <height>1</height>
             </rect>
            </property>
            <property name="frameShape">
              <enum>QFrame::HLine</enum>
            </property>
            <property name="FrameShadow">
              <enum>QFrame::Plain</enum>
            </property>
          </widget>
        </item>

        <item>
          <layout class="QHBoxLayout" name="horizontalLayout1">
            <item>
              <widget class="QPushButton" name="button_OK">
                <property name="text">
                  <string>Parse</string>
                </property>
                <property name="minimumWidth">
                 <number>100</number>
                </property>
                <property name="maximumWidth">
                 <number>100</number>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="button_Cancel">
                <property name="text">
                  <string>Cancel</string>
                </property>
                <property name="minimumWidth">
                 <number>100</number>
                </property>
                <property name="maximumWidth">
                 <number>100</number>
                </property>
              </widget>
            </item>
          </layout>
        </item>
      </layout>
    </widget>
  </widget>
  <resources/>
  <connections/>
</ui>
